src/shared/scrapers/USA/GA/index.js

```diff
@@ -180,19 +181,36 @@ const scraper = {
     'Worth County'
   ],
   async scraper() {
+    const afterChange = datetime.scrapeDateIsAfter('2020-3-26');
```

Comment:
I really like to see separate, completely discrete scrapers instead of logic like this -- it's only sustainable for the first change, once there's a second, it really gets screwy. Also, totally separate scrapers with '2020-3-26: async function() {}` make sure we don't inadvertently mess up scrapers that used to work.